CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## [3.6.0](https://github.com/marklogic-community/mlxprs/releases/tag/v3.6.0)
```

Comment:
Let's adopt the approach of the Java Client here - https://github.com/marklogic/java-client-api/blob/master/CHANGELOG.md - and only store these in the actual Release page. 

I've found this to be really useful for the ml-gradle projects, because it's convenient to be able to edit these without having to make a commit. 

I just created a new 3.6.0 draft release - just paste the good content you have here into that release description.

Reply:
Leaving CHANGELOG.md in place since VSCode, both the marketplace and the application, are expecting it.